### ISSUE TITLE
facing some issue in reactplayer

### ISSUE THREAD

**Issue by maulik225** [maulik225/69fitness#4]
got an error regarding reactplayer package 
I am attaching screenshot for more details
![Capture](https://user-images.githubusercontent.com/40738427/169666326-7e669a0d-96d8-4529-8b85-67392a5759ad.PNG)

**Comment by maulik225** [comment]
Did you face this error @uemrey0 ?

**Comment by uemrey0** [comment]
I see let me check!

**Comment by uemrey0** [comment]
This error was everywhere but I cant find any solution. I've looked up some StackOverflow questions about it and I've read that it's an error coming from the Youtube iFrame API when you make a request for a Youtube video over http instead of https. I put some logging in the react-player library and saw that the requests are supposedly being made over https, so I don't think that's the issue. I realized that this issue happening because of react-player. Provider settled as `https://youtube.com` instead of `http://localhost:3000` which is your url.

This chronic problem does not prevent work as far as I can see.
